### ISSUE TITLE
Titles for anon-ib and re-ripping

### ISSUE THREAD

**Issue by domonic46** [RipMeApp/ripme#197]
* Ripme version: 1.6.10
* Operating system:  windows 7
* Exact URL you were trying to rip when the problem occurred: (NSFW) anon-ib.la/bt/res/20416.html
* Please include any additional information about how to reproduce the problem:

1.  The folder titles from the anon-ib rips don't include the numbers from the url (in this example '20416')  anymore so any thread ripped from one category (in this case the 'bt' category) ends up in the same folder. The previous versions of ripme included the numbers in the folder title.
2. Re-ripping a deleted folder issue:  If I delete the folder created from ripme after ripping anon-ib.la/bt/res/20416.html (or any site supported by ripme - this issue is not exclusive to anon-ib), then rip the same url again, ripme does not seem to rip anything it doesn't generate a folder with any media.

**Comment by cyian-1756** [comment]
> then rip the same url again, ripme does not seem to rip anything it doesn't generate a folder with any media.

This is a feature. If you want to disable it go to the configuration menu and uncheck "Remember URL history"

> The folder titles from the anon-ib rips don't include the numbers from the url (in this example '20416') anymore so any thread ripped from one category (in this case the 'bt' category) ends up in the same folder. The previous versions of ripme included the numbers in the folder title.

I'll get on fixing that

**Comment by cyian-1756** [comment]
@metaprime Looking into this issue I found some pretty questionable code in chanRipper.java

```
@Override
    public boolean canRip(URL url) {
        for (ChanSite _chanSite : explicit_domains) {
            if (_chanSite.domains.contains(url.getHost())) {
                return true;
            }
        }
        return  url.toExternalForm().contains("/res/")     // Most chans
             || url.toExternalForm().contains("/thread/"); // 4chan, archive.moe
    }
```

It looks like chanRipper just claims any url with either /res/ or /thread/ as a chan site. I think it's a pretty bad idea to allow rippers to be this greedy, we don't urls to trigger more than one ripper/the wrong ripper

**Comment by metaprime** [comment]
@cyian-1756 Yeah that does seem pretty sketchy.

**Comment by cyian-1756** [comment]
@domonic46 This has been fixed in 1.6.11
